src/popup/utils/constants.js

```diff
@@ -126,7 +126,6 @@ export const TIP_SERVICE = `${BACKEND_URL}/claim/submit`;
 export const NO_POPUP_AEPPS = [
   'youdonotneedacapetobeahero.com',
   'superhero.com',
-  'localhost',
```

Comment:
I propose to have it enabled only in dev builds because I'm depending on it in https://github.com/aeternity/superhero-wallet/pull/365